src/sscce-sequelize-7.ts

```diff
@@ -30,6 +36,20 @@ export async function run() {
     modelName: 'Foo',
   });
 
+  // You can also use the new decorators model definition
+  class Bar extends Model {
```

Comment:
considering this is the recommended way of using Sequelize 7, I would make this a replacement of Foo instead

Note that you don't need the decorators for the primary key attribute, only the type definition, as it's automatically added

You also need the two generics

```suggestion
  class Bar extends Model<InferAttributes<Bar>, InferCreationAttributes<Bar>> {
```